prusti-viper/src/encoder/high/lower/expression.rs

```diff
@@ -69,6 +69,9 @@ impl IntoPolymorphic<vir_poly::Expr> for vir_high::Expression {
             vir_high::Expression::FuncApp(expression) => {
                 vir_poly::Expr::FuncApp(expression.lower(encoder))
             }
+            vir_high::Expression::BuiltinFuncApp(_expression) => {
+                todo!()
```

Comment:
Would be good to give a proper Unsupported error message here instead of panicking.